harness/ff-react-client-sdk
# FFM-9032 Bump JavaScript SDK version to 1.15.0

# What
 Bump JavaScript SDK version from 1.13.0 to 1.15.0 - see release notes for that version https://github.com/harness/ff-javascript-client-sdk/releases/tag/1.15.0 

# Testing 
Smoke tests on the sample application:
- toggle flags for all flag types, they all update
- turn internet off, observe new SSE disconnect messages, turn it back on 10 seconds later and it reconnects and stream toggles as normal
